client/src/main.ts

```diff
@@ -1,6 +1,7 @@
 import Vue from 'vue';
 import VueCompositionAPI from '@vue/composition-api';
 import Vuetify from 'vuetify';
+import 'polyfill-object.fromentries';
```

Comment:
What does this do?

Reply:
>The pyppeteer chromium version is quite out of date and is missing
`Object.fromEntries`. We use this in a few places so it is easier to
simply add the polyfill than to factor it out.